sematic/db/migrate.py

```diff
@@ -290,7 +290,7 @@ def migrate_down():
     "file",
     type=click.STRING,
     default="schema.sql",
-    help="File to dump the schema to.",
+    help="File into which to dump the new schema.",
```

Comment:
This is the 3rd occurrence of the help message. It is updated to match the wording of the other 2.